app/(tabs)/index.tsx

```diff
@@ -1,51 +1,21 @@
 import { Workouts } from '@/components/Workouts';
 import { View, StyleSheet } from 'react-native';
-import { Button, Overlay, Text, Input, Divider, ListItem } from '@rneui/themed';
+import { Button } from '@rneui/themed';
 import { useState } from 'react';
-import Ionicons from '@expo/vector-icons/Ionicons';
+import { AddWorkoutFormOverlay } from '@/components/AddWorkoutForm';
 
 export default function Tab() {
-  const [visible, setVisible] = useState(false);
-  const [workoutName, setWorkoutName] = useState('');
-  const exercises = ['Deadlift', 'Squat', 'Bench Press'];
+  const [visible, setVisible] = useState<boolean>(false);
 
-  const toggleOverlay = () => {
+  const handleToggleOverlay = () => {
     setVisible(!visible);
```

Comment:
This is fine. But it's better to do:

`setVisible((prev) => !prev)`

The setState callback gives you the previous value